src/auro-flight.js

```diff
@@ -18,11 +18,10 @@ import "./auro-flight-main";
  * Please DO NOT modify unit tests pertaining to DoT regulations without contacting gus@alaskaair.com.
  *
  * @attr {Array} flights - Array of flight numbers `['AS 123', 'EK 432']`
- * @attr {String} duration - String for the duration. `1h 23m`
- * @attr {Number} daysChanged - Number of days changed due to flight duration and timezone. Positive whole integer
- * @attr {String} departureTime - String for the departure time. `9:06 am`
+ * @attr {Number} duration - String for the duration. `505`
+ * @attr {String} departureTime - String for the departure UTC time. `2022-04-13T12:30:00-04:00`
  * @attr {String} departureStation - String for the departure station. `SEA`
- * @attr {String} arrivalTime - String for the arrival time. `4:05 pm`
+ * @attr {String} arrivalTime - String for the arrival UTC time. `2022-04-13T12:30:00-04:00`
```

Comment:
Can we provide more documentation around the desired date format? I'm not sure if "UTC time" is enough. Is this ISO 8601?

Reply:
ok, so there are a couple of issues here. 

1. the auro-datetime element appears to not be correctly converting UTC time, zulu works great by the way. So the issue is within the conversion of local time. 
2. there is missing logic in the calc to span days.